src/api/rpc_api/author.rs

```diff
@@ -106,13 +106,21 @@ pub trait SubmitAndWatch {
 
 	/// Submit an extrinsic and watch it until the desired status
 	/// is reached, if no error is encountered previously.
+	/// The events are not fetched. So no events are listed in the report.
```

Comment:
```suggestion
	/// Associated events are not fetched.
```

Reply:
I think it is important to note that no events are listed in the report.